server/exporter.js

```diff
@@ -15,44 +19,112 @@ const queueOptions = {
   },
 };
 
-async function exportAndEmailCollections(teamId: string, email: string) {
+async function eventCreate(teamId, userId, exportData) {
+  await Event.create({
```

Comment:
We seem to be overloading the `collections.export_all` events here – it's a cool idea to send the progress of the export to the frontend in realtime but I think we can think of those events as just updates of the export/fileoperation model. e.g. `exports.update` / `operation.update` 

Also of note, every `Event.create` is stored in the database and accessible through the audit log, so all of these individual status updates would become an audit log. That's probably overkill, we only need one for the initial export and the rest can be sent over the message bus without being persisted. For that you can use `Event.add` which is the non-persisting alternative.